test/src/kbasesearchengine/test/search/ObjectDataTest.java

```diff
@@ -150,6 +158,10 @@ public void buildMaximal() {
         keyprops.put("null", null);
         keyprops.put("ws", "   \t   \n   ");
         assertThat("incorrect key props", od.getKeyProperties(), is(keyprops));
+        final Map<String, ArrayList> highlight = new HashMap<>();
+        highlight.put("field", new ArrayList<>(Arrays.asList("match")));
+        assertThat("incorrect highlight field", od.getHighlight().keySet(), is(highlight.keySet()));
```

Comment:
Just test the highlight output directly

Reply:
what do you mean?

Reply:
assertThat("message", od.getHighlight(), is(expected));